docs/docusaurus/docs/deployment_patterns/how_to_instantiate_a_data_context_hosted_environments.md

```diff
@@ -2,6 +2,8 @@
 title: Deploying Great Expectations in a hosted environment without file system or CLI
 ---
 
+import CLIRemoval from '/docs/components/warnings/_cli_removal.md'
+
 By default, creating a standard deployment of Great Expectations relies on two components:
 
```

Comment:
The banner didn't appear on this page. And that is because we actually didn't add the CLIRemoval element to this page. Please double check if we've add the element to all the pages.

Reply:
@joshzzheng the `CLIRemoval` element has been added and the admonitions now appear in the documentation.